apps/frontend/src/routes/login/index.tsx

```diff
@@ -97,3 +97,48 @@ export default component$(() => {
     </div>
   );
 });
+
+export const head: DocumentHead = {
+  title: 'Reduced.to | Login',
+  meta: [
+    {
+      name: 'title',
+      content: 'Reduced.to | Login',
+    },
+    {
+      name: 'description',
+      content:
+        'Login to your Reduced.to account to manage your links and QR-Codes.',
```

Comment:
We are not managing QR-Codes in our system.
Change the description to be something like: "Login to your Reduced.to account to manage your shortened links"